### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tcsian**
there was some  English language problem so I fixed that problem and resubmitted that

### CODE REVIEW

**Comment by tcsian** [comment]
Grammatical errors those fixed by me

File: README.md [hunk 1]
```diff
@@ -40,28 +40,28 @@ See [source](examples/example-shader/src/lib.rs) for full details.
 
 ## Background
 
-Historically in games GPU programming has been done through writing either HLSL, or to a lesser extent GLSL. These are simple programming languages that have evolved along with rendering APIs over the years. However, as game engines have evolved, these languages have failed to provide mechanisms for dealing with large codebases, and have generally stayed behind the curve compared to other programming languages.
+Historically in games, GPU programming has been done through writing either HLSL or to a lesser extent GLSL. These are simple programming languages that have evolved along with rendering APIs over the years. However, as game engines have evolved, these languages have failed to provide mechanisms for dealing with large codebases, and have generally stayed behind the curve compared to other programming languages.
```
Comment (termhn) [inline README.md:43]: this comma is incorrect; there should either be no comma like the original or there should also be a comma after Historically.

File: README.md [hunk 2]
```diff
@@ -40,28 +40,28 @@ See [source](examples/example-shader/src/lib.rs) for full details.
...
 ## Project scope
 
-The scope of this overall project is quite broad, but is in multiple stages
+The scope of this overall project is quite broad but is in multiple stages
 
 - `rustc` compiler backend to generate [SPIR-V], plugging in via `-Z codegen-backend`. 
   - This is the same mechanism that [rustc_codegen_cranelift](https://github.com/bjorn3/rustc_codegen_cranelift) and [rustc_codegen_gcc](https://github.com/antoyo/rustc_codegen_gcc) use.
-  - Currently only [SPIR-V] support is planned, [Vulkan](https://en.wikipedia.org/wiki/Vulkan_(API))'s open compiler target
-  - Possible a future version could suppport [DXIL](https://github.com/microsoft/DirectXShaderCompiler/blob/master/docs/DXIL.rst) (the target for DirectX) or [WGSL](https://github.com/gpuweb/gpuweb/tree/main/wgsl) (the WebGPU shading language that's bijective with SPIR-V)
+  - Currently, only [SPIR-V] support is planned, [Vulkan](https://en.wikipedia.org/wiki/Vulkan_(API))'s open compiler target
```
Comment (termhn) [inline README.md:63]: Should just restructure this sentence to "Currently, support is planned only for SPIR-V, Vulkan's open compiler target"

**Review by termhn — CHANGES_REQUESTED** [review]

**Comment by XAMPPRocky** [comment]
Going to close this as there hasn't been any activity in a couple of months and it needs to be rebased. Feel free to open a PR!

**Pull request closed without merging**